apps/dialtone-documentation/docs/components/hovercard.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Hovercard
+description: The Hovercard component provides a popover-like experience for displaying additional information.
```

Comment:
"A Hovercard toggles a content overlay when its anchor element is hovered for a minimum amount of time."